test-framework/junit5/src/main/java/org/apache/camel/quarkus/test/CamelQuarkusTestSupport.java

```diff
@@ -199,6 +199,7 @@ protected void bindToRegistry(Registry registry) throws Exception {
     /**
      * The same functionality as {@link CamelTestSupport#postProcessTest()} .
      */
+    @Deprecated(since = "4.7.0")
```

Comment:
Since `CamelQuarkusTestSupport` lives in this project. I think we should use our versioning. So for all deprecated methods:

```suggestion
    @Deprecated(since = "3.15.0")
```

Reply:
fixed